source/gui/settingsDialogs.py

```diff
@@ -2700,6 +2700,42 @@ def __init__(self, parent):
 		self.ariaDescCheckBox.SetValue(config.conf["annotations"]["reportAriaDescription"])
 		self.ariaDescCheckBox.defaultValue = self._getDefaultValue(["annotations", "reportAriaDescription"])
 
+		# Translators: This is the label for a group of advanced options in the
+		#  Advanced settings panel
+		label = _("HID Braille Standard")
+		hidBrailleSizer = wx.StaticBoxSizer(wx.VERTICAL, self, label=label)
+		hidBrailleBox = hidBrailleSizer.GetStaticBox()
+		hidBrailleGroup = guiHelper.BoxSizerHelper(self, sizer=hidBrailleSizer)
+		self.bindHelpEvent("HIDBraille", hidBrailleBox)
+		sHelper.addItem(hidBrailleGroup)
+
+		supportHidBrailleChoices = [
+			# Translators: Label for option in the 'Enable support for HID braille' combobox
+			# in the Advanced settings panel.
+			_("Default (Yes)"),
+			# Translators: Label for option in the 'Enable support for HID braille' combobox
+			# in the Advanced settings panel.
+			_("Yes"),
+			# Translators: Label for option in the 'Enable support for HID braille' combobox
+			# in the Advanced settings panel.
+			_("No"),
+		]
+
+		# Translators: This is the label for a checkbox in the
+		#  Advanced settings panel.
+		label = _("Enable support for HID braille")
+		self.supportHidBrailleCheckbox: wx.Choice = hidBrailleGroup.addLabeledControl(

```

Comment:
Here and at other places, including comments:
Checkbox -> Combo

```suggestion
		self.supportHidBrailleCombo: wx.Choice = hidBrailleGroup.addLabeledControl(

```